--- NeJE\dlg\DHROTH.D	Sat Feb 03 03:00:26 2018
+++ C:\BWP Patchstudio\patched files\NeJE\dlg\DHROTH.D	Sun Sep 15 19:22:31 2019
@@ -325,7 +325,7 @@
 IF ~~ THEN BEGIN 45
   SAY ~And please talk to Accalia. She is an initiate at the temple of Tempus. I have recently spoken with her. She is willing to join our expedition.~ [HROT_46]
   IF ~~ THEN REPLY ~Very well. I shall return shortly.~ DO ~SetGlobal("Supply_Quest","GLOBAL",1)
-GiveItemCreate("List",Protagonist,1,0,0)
+GiveItemCreate("List",Player1,1,0,0)
 SetGlobal("Join_Accalia","GLOBAL",1)~ EXIT
 END
 
