--- c#husam\crossmod\husam_alternatives.d	Tue Apr 11 18:28:02 2023
+++ C:\BWP Patchstudio\patched files\c#husam\crossmod\husam_alternatives.d	Sun Jun 11 17:35:18 2023
@@ -109,7 +109,7 @@
 
 IF ~~ THEN malficus_path_02
 SAY ~Nevertheless, I will go with you. Why? Because I am convinced that if you don't go to Irenicus, he will come for you. And that is all that matters.~
-IF ~~ THEN DO ~Global("C#Husam2_CM_AltMal","GLOBAL",2)~ EXIT
+IF ~~ THEN DO ~SetGlobal("C#Husam2_CM_AltMal","GLOBAL",2)~ EXIT
 END
 
 
