--- BG1NPCSoA\dialogues\7XAJAN.d	Mon Feb 06 08:44:26 2023
+++ C:\BWP Patchstudio\patched files\BG1NPCSoA\dialogues\7XAJAN.d	Wed Aug 02 15:33:18 2023
@@ -1127,7 +1127,7 @@
 IF WEIGHT #-70 ~Global("AjantisQuest","GLOBAL",9)~ THEN BEGIN 7xAjantisQuestTalk-4
 SAY @393
 IF ~~ THEN DO ~SetGlobal("AjantisQuest","GLOBAL",10) SetGlobalTimer("7xAjantisQuestTimer2","GLOBAL",ONE_DAY)~ REPLY @394 GOTO 7xAjantisQuestTalk-5
-IF ~~ THEN DO ~SetGlobal("AjantisQuest","GLOBAL",99) EraseJournalEntry(@1850,QUEST) EraseJournalEntry(@1849,QUEST)~ REPLY @404 GOTO 7xAjantisQuestTalk-4-End
+IF ~~ THEN DO ~SetGlobal("AjantisQuest","GLOBAL",99) EraseJournalEntry(@1850) EraseJournalEntry(@1849)~ REPLY @404 GOTO 7xAjantisQuestTalk-4-End
 END
 
 IF ~~ THEN BEGIN 7xAjantisQuestTalk-5
