--- JklHel\dialogue\jkl_hel_partyactions.D	Fri Jul 02 14:11:02 2021
+++ C:\BWP Patchstudio\patched files\JklHel\dialogue\jkl_hel_partyactions.D	Fri Jul 02 18:19:24 2021
@@ -95,7 +95,7 @@
 THEN JKLHELP LeaveParty
 ~I'll not remain if you feel I'm not needed... I'll return to the Friendly Arm Inn. If you need my help again, you may find me there.~
 DO ~SetGlobal("JKLHEL_Fired","GLOBAL",1)
-EscapeAreaMove("AR6801",755,390,SW)~ // copied from Neera, might need to get new coordinates
+EscapeAreaMove("AR6801",755,390,2)~ // copied from Neera, might need to get new coordinates
 EXIT
 
 CHAIN
