--- stratagems\stratagems.ini	Thu Sep 16 07:46:04 2021
+++ C:\BWP Patchstudio\patched files\stratagems\stratagems.ini	Thu Jun 01 09:32:59 2023
@@ -39,14 +39,14 @@
 
 [nerfed and moved items]
 
-no_item_nerf_balduran_shield=1
-no_item_nerf_magi_staff=0
-no_item_nerf_mirror_cloak = 0
-no_item_nerf_carsomyr = 0
-no_item_nerf_resurrection_rod = 0
-no_item_nerf_vhailor_helm = 0
-no_item_nerf_vecna = 0
-no_initial_change_elemental_arrows = 0
+no_item_nerf_balduran_shield = 1
+no_item_nerf_magi_staff = 1
+no_item_nerf_mirror_cloak = 1
+no_item_nerf_carsomyr = 1
+no_item_nerf_resurrection_rod = 1
+no_item_nerf_vhailor_helm = 1
+no_item_nerf_vecna = 1
+no_initial_change_elemental_arrows = 1
 
 [component tweaks]
 
