--- FotD\BAF_NEW\AR1010.baf	Thu Apr 21 21:20:36 2005
+++ C:\BWP Patchstudio\patched files\FotD\BAF_NEW\AR1010.baf	Wed Nov 30 07:12:57 2016
@@ -5,12 +5,12 @@
   RESPONSE #100
     Wait(10)
     SetGlobal("iceisle","GLOBAL",1)
-    ActionOverride(Player1,LeaveAreaLUAPanic("AR1000","",[1798.624],12))
-    ActionOverride(Player1,LeaveAreaLUA("AR1000","",[1798.624],12))
-    ActionOverride(Player2,LeaveAreaLUA("AR1000","",[1793.748],12))
-    ActionOverride(Player3,LeaveAreaLUA("AR1000","",[1736.751],12))
-    ActionOverride(Player4,LeaveAreaLUA("AR1000","",[1681.726],12))
-    ActionOverride(Player5,LeaveAreaLUA("AR1000","",[1681.726],12))
-    ActionOverride(Player6,LeaveAreaLUA("AR1000","",[1795.687],12))
+    ActionOverride(Player1,LeaveAreaLUAPanic("%AR1000%","",[1798.624],12))
+    ActionOverride(Player1,LeaveAreaLUA("%AR1000%","",[1798.624],12))
+    ActionOverride(Player2,LeaveAreaLUA("%AR1000%","",[1793.748],12))
+    ActionOverride(Player3,LeaveAreaLUA("%AR1000%","",[1736.751],12))
+    ActionOverride(Player4,LeaveAreaLUA("%AR1000%","",[1681.726],12))
+    ActionOverride(Player5,LeaveAreaLUA("%AR1000%","",[1681.726],12))
+    ActionOverride(Player6,LeaveAreaLUA("%AR1000%","",[1795.687],12))
 END
 
