--- PoFQuestPack\English\D\ATKIMOD1.tra	Mon Mar 16 17:18:28 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\ATKIMOD1.tra	Sun Mar 06 16:48:46 2022
@@ -6,5 +6,5 @@
 @5 = ~ Business intersection.
 
 I agreed that temporarily me and my company will act as the personal guard of trade king Amn. ~
-@6 = ~ Forget it, I won’t cut heads for 5000 gold, find someone else. ~
+@6 = ~ Forget it, I won't cut heads for 5000 gold, find someone else. ~
 @7 = ~ I was counting on you, but this is your decision. Goodbye. ~
