--- PoFQuestPack\English\D\BG1MOD.tra	Mon Mar 16 17:22:22 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\BG1MOD.tra	Sun Mar 06 16:48:46 2022
@@ -4,5 +4,5 @@
 @4 = ~ Thank you. Come again. ~
 @5 = ~ What do you want? ~
 @6 = ~ I'm looking for a killer who is responsible for the 5th killed people, they told me that he was last seen here. ~
-@7 = ~ I'm sorry, but I don’t know anything about it, try asking visitors. ~
+@7 = ~ I'm sorry, but I don't know anything about it, try asking visitors. ~
 @8 = ~ Good, thanks anyway. ~
