--- PoFQuestPack\English\D\BGGMOD.tra	Mon Mar 16 17:22:22 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\BGGMOD.tra	Sun Mar 06 16:48:46 2022
@@ -1,4 +1,4 @@
-@1 = ~ Don’t kill me, please! ~
+@1 = ~ Don't kill me, please! ~
 @2 = ~ I did not come here to kill you. What are you doing here? ~
 @3 = ~ Someone kidnapped me and keeps me here. ~
 @4 = ~ You are free girl. Can you get to the captain of the city guard yourself? ~
