--- PoFQuestPack\English\D\BGPS2MOD.tra	Mon Mar 16 17:22:22 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\BGPS2MOD.tra	Sun Mar 06 16:48:46 2022
@@ -1,4 +1,4 @@
-@1 = ~ I did nothing, what’s the matter? ~
+@1 = ~ I did nothing, what's the matter? ~
 @2 = ~ I'm looking for the killer responsible for the deaths of five people. ~
 @3 = ~ I have nothing to do with it, but for 100 gold, I can help. ~
 @4 = ~ That's 100 gold, speak immediately! ~
