--- PoFQuestPack\English\D\CLC4MOD.tra	Mon Mar 16 18:28:56 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\CLC4MOD.tra	Sun Mar 06 16:48:47 2022
@@ -1,7 +1,7 @@
 @0 = ~ Welcome to the sanctuary of our god Cyric, built largely thanks to you, <CHARNAME>. Good to see you again. ~
 @1 = ~ And I am glad, high priest. Why did you send for me? ~
 @2 = ~ Sorry <CHARNAME> that I am bothering you again, but I have one assignment. Torm priests poison our lives again, and at this moment you are the only one who can save us from them once and for all. ~
-@3 = ~ I'm really sorry, but right now I don’t have time to work for you. Maybe later. ~
+@3 = ~ I'm really sorry, but right now I don't have time to work for you. Maybe later. ~
 @4 = ~ And what is it? ~
 @5 = ~ Far to the north of here in the frozen plains is an island, and under it should be a very valuable historical scroll. A hidden secret about the confrontation between Torm and Cyric. And yet - we learned that admirers of Torm sent their group of adventurers to find this scroll. ~
 @6 = ~ And what do we need to do for the high priest? ~
