--- PoFQuestPack\English\D\DRAG1MOD.tra	Mon Mar 16 18:32:22 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\DRAG1MOD.tra	Sun Mar 06 16:48:47 2022
@@ -7,7 +7,7 @@
 @6 = ~ For me, dragons are enemies too, die a monster! ~
 @7 = ~ These thieves are very skilled, they stole my powerful magic wand, which I had to guard. This task was assigned to me by a very powerful wizard. I have to get it back at all costs before the master returns. ~
 @8 = ~ I can help you get that rod back, provided I get a reward in return. ~
-@9 = ~ What happened to your thing will dishonor your name, but I don’t have time for that. I'm leaving. ~
+@9 = ~ What happened to your thing will dishonor your name, but I don't have time for that. I'm leaving. ~
 @10 = ~ If you succeed, a very valuable item, a gift from the gods, will be waiting for you, so do you agree <CHARNAME>? ~
 @11 = ~ I will find him. ~
 @12 = ~ Dragon artifact.
