--- PoFQuestPack\English\D\DRB1MOD.tra	Mon Mar 16 18:32:22 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\DRB1MOD.tra	Sun Mar 06 16:48:47 2022
@@ -20,5 +20,5 @@
 @15 = ~ Sylvanowed.
 
 You managed to turn everyone against yourself, both the drow from Ched Nasad and the drow priestesses of Eilistraee. The answers to your questions can no longer be found. ~
-@16 = ~ Hmm ... You are back, my scouts have discovered a dead dragon. Excellent job adventurers. I forgive you. So, back to why you came. I heard about the incident, you can be thankful to our sister for saving her life. Well, could you help us again a little, but don’t worry, there won’t be any kills of dragons or different demons. ~
+@16 = ~ Hmm ... You are back, my scouts have discovered a dead dragon. Excellent job adventurers. I forgive you. So, back to why you came. I heard about the incident, you can be thankful to our sister for saving her life. Well, could you help us again a little, but don't worry, there won't be any kills of dragons or different demons. ~
 @17 = ~ The dragon is still alive, <CHARNAME>. Kill him! ~
