--- PoFQuestPack\English\D\DUERLMOD.tra	Mon Mar 16 18:32:22 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\DUERLMOD.tra	Sun Mar 06 16:48:47 2022
@@ -8,7 +8,7 @@
 @7 = ~ Mine.
 
 The duergar leader will give me two valuable items if I get rid of the gnomes in the central part of the mine. ~
-@8 = ~ I heard that after Elgen’s death, most of the dwarves escaped from here ... Great job. Here is the promised reward. ~
+@8 = ~ I heard that after Elgen's death, most of the dwarves escaped from here ... Great job. Here is the promised reward. ~
 @9 = ~ Thanks and goodbye. ~
 @10 = ~ Mine.
 
