--- PoFQuestPack\English\D\DWQMOD.tra	Mon Mar 16 18:32:22 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\DWQMOD.tra	Sun Mar 06 16:48:47 2022
@@ -1,4 +1,4 @@
-@0 = ~ Who are you? It was not me who did it! This is a misunderstanding, I didn’t kill anyone! ~
+@0 = ~ Who are you? It was not me who did it! This is a misunderstanding, I didn't kill anyone! ~
 @1 = ~ I <CHARNAME>, your mother sent me. Here is her pendant so you believe me. ~
 @2 = ~ Wait, you were at my initiation. I remember you. Why don't we join forces? I want to learn something, you look like you went through a lot. I am sure that I will be useful to many people, and under your protection I will be calmer if I join your party, What do you say? ~
 @3 = ~ I think everyone should do their own thing. Each of us has his own task. ~
