--- PoFQuestPack\English\D\ELADMOD.tra	Mon Mar 16 18:36:28 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\ELADMOD.tra	Sun Mar 06 16:48:47 2022
@@ -11,7 +11,7 @@
 
 An elf named Eladriel told me about a mysterious forest near Neverwinter, he wants me to find a source of dark fog that envelops the forest. ~
 @11 = ~ Okay, we'll find out what it is. ~
-@12 = ~ Now I don’t have time, I will come back later. ~
+@12 = ~ Now I don't have time, I will come back later. ~
 @13 = ~ Great, brilliant work, I saw when the fog around the forest disappeared. So what was that? ~
 @14 = ~ In the dungeon under the forest, the Lord of the undead hid. He created a magical fog around the forest, which weakened people and strengthened dark creatures. They hunted them, and then with the help of magic they turned their own kind into themselves. I stopped all this and the forest is safe again. ~
 @15 = ~ Thank you stranger, as I promised, here is something for you. When I was young, like you traveled a lot. Take this sword, it saved my life many times. I don't need him anymore. Well, I need to go do my own thing. Bye. ~
