--- PoFQuestPack\English\D\ELGMOD.tra	Mon Mar 16 18:36:28 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\ELGMOD.tra	Sun Mar 06 16:48:47 2022
@@ -4,7 +4,7 @@
 @3 = ~ I <CHARNAME>, some dwarf sent me to you, he said that you have problems. ~
 @4 = ~ We make mithril weapons and armor here, but recently duergar appeared here and settled in the eastern part of the mine. They want to get mithril for themselves. ~
 @5 = ~ Why not send a message about the situation to other dwarves or their rulers? ~
-@6 = ~ I don’t want to lose my job, they’ll probably replace me with someone else. The problem is that there has always been a war between grays and dwarves, and that cannot be changed! I wanted to say that we cannot negotiate with them, we must destroy them. What if you helped us? ~
+@6 = ~ I don't want to lose my job, they'll probably replace me with someone else. The problem is that there has always been a war between grays and dwarves, and that cannot be changed! I wanted to say that we cannot negotiate with them, we must destroy them. What if you helped us? ~
 @7 = ~ Yes, I will help you, what do I need to do? ~
 @8 = ~ Duergar at the entrance advised me not to make a mistake and not to go into the wrong. And I agree with him, good luck and goodbye! ~
 @9 = ~ There are 3 paths leading from this room, one on which you came, another on the east leads to the shelter of the grays, and the last western corridor will lead to monsters, which also need to be destroyed. Find and kill their leaders and bring me their heads. ~
