--- PoFQuestPack\English\D\FIRQMOD.tra	Mon Mar 16 18:45:46 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\FIRQMOD.tra	Sun Mar 06 16:48:47 2022
@@ -1,4 +1,4 @@
-@1 = ~ Hello! I have a riddle for you, if you can guess, I will teleport you closer to what you are looking for, if you don’t get it, then you will never pass. If you succeed, you teleport closer. ~
+@1 = ~ Hello! I have a riddle for you, if you can guess, I will teleport you closer to what you are looking for, if you don't get it, then you will never pass. If you succeed, you teleport closer. ~
 @2 = ~ So this is the riddle of the demon. ~
 @3 = ~ Although I have been given a form, I myself am nothing,
 there are a thousand things in me.
@@ -15,7 +15,7 @@
 here is the beggar's clothes on me.
 Who was a huge ogre, now an ordinary goblin,
 jsem kdekdo, nemam vsak svuj vzhled.
-I don’t feel sadness and joy again
+I don't feel sadness and joy again
 lips move, but not enough voice.
 I will not die, since I have not yet been born.
 So please tell me - who am I? ~
