--- PoFQuestPack\English\D\FNGRADAK.tra	Mon Mar 16 18:45:46 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\FNGRADAK.tra	Sun Mar 06 17:46:51 2022
@@ -1,3 +1,3 @@
 @1 = ~ We met again <CHARNAME>, I'm glad you were here when your friend Grafalcon died from my sword! ~
 @2 = ~ This fight was not fair Gradak! Your magicians have weakened his strength! Whenever we met, you always managed to slip away ... This time, you have nowhere to go. I will avenge Grafalkon and all those whom you killed. ~
-@3 = ~ You ruined my plan to capture Faeríàn <CHARNAME>! You will die for it! ~
+@3 = ~ You ruined my plan to capture Faerun <CHARNAME>! You will die for it! ~
