--- PoFQuestPack\English\D\GLVMOD.tra	Mon Mar 16 18:48:36 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\GLVMOD.tra	Sun Mar 06 17:46:55 2022
@@ -1,8 +1,8 @@
 @0 = ~ Greetings to you adventurers. Perhaps you are interested in serious fights or you want to earn some money? ~
 @1 = ~ Of course, can you tell in more detail? ~
-@2 = ~ This is a gladiatorial battle in the arena, in the city of nomads, near the Anauroch desert. The best warriors of Faeríàn fight there. If you are interested, I will sell you the card on which this place is indicated, for only 1000 gold. ~
+@2 = ~ This is a gladiatorial battle in the arena, in the city of nomads, near the Anauroch desert. The best warriors of Faerun fight there. If you are interested, I will sell you the card on which this place is indicated, for only 1000 gold. ~
 @3 = ~ Okay, here you go. ~
-@4 = ~ I don’t have so much with me, I will come back later. ~
+@4 = ~ I don't have so much with me, I will come back later. ~
 @5 = ~ I'm not interested in gladiatorial battles. ~
 @6 = ~ Thank you, here is a map showing how to get to the arena. ~
 @7 = ~ Thank you too. Goodbye. ~
