--- PoFQuestPack\English\D\H1MOD.tra	Mon Mar 16 18:51:10 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\H1MOD.tra	Sun Mar 06 16:48:47 2022
@@ -1,6 +1,6 @@
 @0 = ~ This is the cave where the demon is! ~
 @1 = ~ Is he at the entrance or in the back of the cave? ~
-@2 = ~ I don’t know for sure, but you will probably meet him there. ~
+@2 = ~ I don't know for sure, but you will probably meet him there. ~
 @3 = ~ Okay, then goodbye. ~
 @4 = ~ Hrothgar.
 
