--- PoFQuestPack\English\D\HCMOD.tra	Mon Mar 16 18:51:10 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\HCMOD.tra	Mon Mar 07 17:41:25 2022
@@ -1,6 +1,6 @@
 @1 = ~ <CHARNAME>, I knew there would be complications when Elminster announced that he was inviting you to defend the ceremony. ~
-@2 = ~ Tassili, do you really think that you with Gradak and a bunch of southerners can conquer all of Faerín? ~
+@2 = ~ Tassili, do you really think that you with Gradak and a bunch of southerners can conquer all of Faerun? ~
 @3 = ~ If Gradak receives these 3 relics and unites the mages, even a handful can reach the goal. ~
 @4 = ~ If I kill you now Tacillo, Gradac will have no one to unite with. ~
-@5 = ~ Do you think I'm scared of you <CHARNAME>? I heard a lot about you, but as I understand it, you have not heard about me, so underestimate me, we’ll also see who is who ... ~
+@5 = ~ Do you think I'm scared of you <CHARNAME>? I heard a lot about you, but as I understand it, you have not heard about me, so underestimate me, we'll also see who is who ... ~
 @6 = ~ You will pay for betrayal! ~
