--- PoFQuestPack\English\D\HROT2MOD.tra	Mon Mar 16 18:51:10 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\HROT2MOD.tra	Sun Mar 06 16:48:47 2022
@@ -4,7 +4,7 @@
 @3 = ~ Did you manage to find out something? What do you want from me? ~
 @4 = ~ I think our priest may know something, go and talk to him in the temple, this is south of here. ~
 @5 = ~ Why is this book so important to Khrodgar? ~
-@6 = ~ I really don’t know this <CHARNAME>, but still I ask you to help me again. ~
+@6 = ~ I really don't know this <CHARNAME>, but still I ask you to help me again. ~
 @7 = ~ I'm sorry, but now I already have other things to do. I will try to help you later. ~
 @8 = ~ Of course, I will go and talk with the priest. ~
 @9 = ~ Oh, I almost forgot. You will need a key to get to the temple. After that sad event, the temple is closed to the public. And I have to go, I have a lot to do. ~
