--- PoFQuestPack\English\D\HROTHMOD.tra	Mon Mar 16 18:51:10 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\HROTHMOD.tra	Sun Mar 06 17:46:55 2022
@@ -2,7 +2,7 @@
 @2 = ~ Who are you, I know you? ~
 @3 = ~ I am Hrothgar, mayor of East Haven, and I loved adventures. ~
 @4 = ~ Never heard of you, I ... ~
-@5 = ~ I know who you are, you <CHARNAME> famous hero, protector of Faeríàn, demigod, son of the Lord of Destruction, Baal’s child and friend of Horion ... I know you well, I hoped you would come and help me. ~
+@5 = ~ I know who you are, you <CHARNAME> famous hero, protector of Faerun, demigod, son of the Lord of Destruction, Baal's child and friend of Horion ... I know you well, I hoped you would come and help me. ~
 @6 = ~ So right to the point, what do you need Hrothgar? ~
 @7 = ~ Wait, did you know Horion? How is it that you know so much about me and my life? Tell me, Hrothgar. ~
 @8 = ~ A month ago I was called to Luskan to accompany a group of dignitaries ~
