--- PoFQuestPack\English\D\KELROTH.tra	Mon Mar 16 19:39:28 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\KELROTH.tra	Sun Mar 06 17:46:56 2022
@@ -4,14 +4,14 @@
 @4 = ~ Was he scared of me or just ran away? Why did you stay here Kelrot? ~
 @5 = ~ I do not think he escaped, he left me here for the chief. ~
 @6 = ~ It seems to me that you were used to proto. Tell me everything you know and I will spare you. ~
-@7 = ~ If you are looking for information, then go to the Dark Forest in northern Faeríàn. It's a long way, but I think it's worth it. ~
+@7 = ~ If you are looking for information, then go to the Dark Forest in northern Faerun. It's a long way, but I think it's worth it. ~
 @8 = ~ You mean that I will find Gradak in the Dark Forest? ~
 @9 = ~ I did not say that you will find Gradak there. I said that you will find information there. ~
 @10 = ~ Why are you saying this to me? If you're here to stop me. ~
 @11 = ~ Yes, we will fight, but I must say that some of your friends were my friends. ~
 @12 = ~ And who is this? Speak! ~
 @13 = ~ Once, Alvin was my student, and I was his teacher. I must say that he became a great magician. But today the strongest will win in our battle. ~
-@14 = ~ Paladins of Faeríàn.
+@14 = ~ Paladins of Faerun.
 
 Kelrot, a former teacher and friend of Elvin, turned out to be serving Gradak and his task was to stop me. Nevertheless, he gave me advice to go to the Dark Forest in northern Faerun. There I can find answers to some of my questions. ~
 @15 = ~ You don't need to fight me, Kelrot! ~
