--- PoFQuestPack\English\D\KUMOD.tra	Mon Mar 16 19:39:28 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\KUMOD.tra	Sun Mar 06 16:48:48 2022
@@ -10,4 +10,4 @@
 @9 = ~ Food.
 
 A cook in Siluvaneda asked me to deliver food to a guard in the west. ~
-@10 = ~ I will have to find someone more intelligent. It’s impossible to solve a business proposal with you! ~
+@10 = ~ I will have to find someone more intelligent. It's impossible to solve a business proposal with you! ~
