--- PoFQuestPack\English\D\LOFLTMOD.tra	Mon Mar 16 19:39:28 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\LOFLTMOD.tra	Sun Mar 06 16:48:48 2022
@@ -5,7 +5,7 @@
 @4 = ~ Pirate Island.
 
 The task was completed, the leader of the stray daggers will not be more likely to create problems, but Aran's friend is unfortunately dead. ~
-@5 = ~ Give me a dagger and 15,000 gold and I will leave you alive. And don’t disturb the village your people recently attacked. ~
+@5 = ~ Give me a dagger and 15,000 gold and I will leave you alive. And don't disturb the village your people recently attacked. ~
 @6 = ~ Okay, this is a wise decision. Here you go. ~
 @7 = ~ Pirate Island.
 
