--- PoFQuestPack\English\D\MILCMOD.tra	Mon Mar 16 19:42:58 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\MILCMOD.tra	Sun Mar 06 17:46:56 2022
@@ -26,13 +26,13 @@
 I found the brother of one of the victims. He told me that before his death, his brother met with a man in the house of the Grand Duke. ~
 @19 = ~ Killer.
 
-I ran into killer Artemis in the city’s sewer. ~
+I ran into killer Artemis in the city's sewer. ~
 @20 = ~ Killer.
 
-I didn’t have the courage to fight the best killer of Faeríàn. ~
+I didn't have the courage to fight the best killer of Faerun. ~
 @21 = ~ Thank you - and goodbye. ~
 @22 = ~ Is there any other work for me? ~
-@23 = ~ It’s very a pity, many people in the city would like to know who killed their relatives, but anyway, you completed your task. Thank you, here is your reward. ~
+@23 = ~ It's very a pity, many people in the city would like to know who killed their relatives, but anyway, you completed your task. Thank you, here is your reward. ~
 @24 = ~ Killer.
 
 The guild of thieves hiding beneath the city is now over. But I missed Artemis. ~
