--- PoFQuestPack\English\D\MONORMOD.tra	Mon Mar 16 19:42:58 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\MONORMOD.tra	Sun Mar 06 16:48:48 2022
@@ -2,7 +2,7 @@
 @2 = ~ Nice to meet you, what do you want? ~
 @3 = ~ My master sent me to find experienced adventurers who could help us in our temple. ~
 @4 = ~ Where is your temple located? And what happened? ~
-@5 = ~ I don’t know exactly what is going on, our teacher is a very wise wizard, he said that we ourselves can’t do it. The temple is located near the border with Kalimshan. Can you help us? You will receive a reward for this. ~
+@5 = ~ I don't know exactly what is going on, our teacher is a very wise wizard, he said that we ourselves can't do it. The temple is located near the border with Kalimshan. Can you help us? You will receive a reward for this. ~
 @6 = ~ Yes, I will visit your teacher. ~
 @7 = ~ Here, I marked the temple on your map, good luck. Bye. ~
 @8 = ~ Forbidden Temple.
