--- PoFQuestPack\English\D\MTRMOD.tra	Mon Mar 16 19:42:58 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\MTRMOD.tra	Sun Mar 06 16:48:48 2022
@@ -2,7 +2,7 @@
 @1 = ~ Speak. ~
 @2 = ~ I am a merchant from Memnon and I am looking for a group of mercenaries or adventurers who are ready to help me protect my goods on the way from Atkatla da Memnon. The road is very dangerous, a meeting with the pirates is very likely. When we deliver the goods to Memnon, you will receive a very valuable artifact, and 6000 gold. Take it? ~
 @3 = ~ Okay, I will guard your goods on the way to Memnon. When will we move out? ~
-@4 = ~ I don’t have time for this merchant now. ~
+@4 = ~ I don't have time for this merchant now. ~
 @5 = ~ It depends on you, if you are ready, then I will take you to the captain of the ship. ~
 @6 = ~ I have something else to do and buy some things for the trip. Then I'll be back. ~
 @7 = ~ Okay, we can go. ~
