--- PoFQuestPack\English\D\MYESMOD.tra	Mon Mar 16 19:42:58 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\MYESMOD.tra	Sun Mar 06 16:48:48 2022
@@ -15,7 +15,7 @@
 @14 = ~ Sarevok? And I thought you already killed him. ~
 @15 = ~ I thought so too. But someone returned him, and his task was to get the magic stone in your city. ~
 @16 = ~ Anyway <CHARNAME>, thanks for the help. Is there anything I can do for you? ~
-@17 = ~ Yes, I’m looking for a strange magician, since many of you stop, maybe you came across him? ~
+@17 = ~ Yes, I'm looking for a strange magician, since many of you stop, maybe you came across him? ~
 @18 = ~ Yes, I know who you are talking about, he went here a few days ago, he asked me about housing, but given our situation, I sent him to a village not far from here. Show your card, I will mark this place on it. ~
 @19 = ~ Thank you priest. Good luck. ~
 @20 = ~ Shadow Lord.
