--- PoFQuestPack\English\D\OLDMMOD.tra	Mon Mar 16 19:53:46 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\OLDMMOD.tra	Sun Mar 06 17:46:56 2022
@@ -3,7 +3,7 @@
 @3 = ~ Yes, yes, I will concentrate on this old man. It will take some time. ~
 @4 = ~ ... ~
 @5 = ~ Uh, yes, I see him. Quarill is currently in a frozen forest, this is the place on your map, and now I apologize, I need to relax a bit, goodbye and good luck. ~
-@6 = ~ Paladins of Faeríàn.
+@6 = ~ Paladins of Faerun.
 
 The old magician was really peculiar, but he fulfilled his task and found the place where Qvarill abducted by Gradak is located. ~
 @7 = ~ Thanks ... ~
