--- PoFQuestPack\English\D\PAL1MOD.tra	Mon Mar 16 19:53:46 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\PAL1MOD.tra	Sun Mar 06 16:48:48 2022
@@ -7,7 +7,7 @@
 @6 = ~ Hrothgar.
 
 The Paladin from Deep Water asked me for help. It is necessary to save the already dead hero from East Harbor. His name is Hrothgar and his followers have found a way to bring him back to life. But, unfortunately, the situation suddenly became complicated and a demon appeared instead of Hrothgar. If you kill the demon, Hrothgar will again be among the living. ~
-@7 = ~ I'm really sorry, but don’t believe all the rumors. Goodbye. ~
+@7 = ~ I'm really sorry, but don't believe all the rumors. Goodbye. ~
 @8 = ~ Hrothgar.
 
 Job rejected. ~
