--- PoFQuestPack\English\Baf_new\PF0153.tra	Mon Mar 16 20:31:48 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\Baf_new\PF0153.tra	Sun Mar 06 18:08:07 2022
@@ -1,3 +1,3 @@
 @0 = ~ I see that Alvin has done his job. Good to see you again <CHARNAME>. ~
 @1 = ~ I am also pleased to see you. But, as you probably know, I have little time, so what is my task? ~
-@2 = ~ Alvin had to tell you something, and you will find out the rest from the Supreme Magician of the council. He is the founder of the Faeríàn Paladin cult. ~
+@2 = ~ Alvin had to tell you something, and you will find out the rest from the Supreme Magician of the council. He is the founder of the Faerun Paladin cult. ~
