--- PoFQuestPack\English\Baf_new\PF1234.tra	Fri Jun 11 11:35:47 2021
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\Baf_new\PF1234.tra	Sun Mar 06 18:08:07 2022
@@ -11,4 +11,4 @@
 @10 = ~ Try to create it again after a while? ~
 @11 = ~ Maybe <CHARNAME>. Maybe sometime in the future. We may need your help again. But in the end, you did a great job. I have a reward for you. During my travels, I collected five very powerful artifacts. Now they are yours. ~
 @12 = ~ Thanks again for helping you and your friends. Now I am teleporting you to Brinnlow, I know that you are looking for an elven sorcerer. So, I wish you good luck. Bye. ~
-@13 = ~ Paladins of Faeríàn. The story ends. Hradak, who betrayed his two friends Alvin and Grafalkon, is now dead, along with the hero Grafalkon. A new order of the Knights of Faeríàn was not created. And as Alvin said, it may never arise because there is not a single person, elf, or anyone else who would not want to gain such power that he could conquer the whole world. ~
+@13 = ~ Paladins of Faerun. The story ends. Hradak, who betrayed his two friends Alvin and Grafalkon, is now dead, along with the hero Grafalkon. A new order of the Knights of Faerun was not created. And as Alvin said, it may never arise because there is not a single person, elf, or anyone else who would not want to gain such power that he could conquer the whole world. ~
