--- PoFQuestPack\English\D\PRTC2MOD.tra	Mon Mar 16 19:53:46 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\PRTC2MOD.tra	Sun Mar 06 16:48:48 2022
@@ -1,7 +1,7 @@
 @0 = ~ Welcome to the shrine of our god Torm, built largely thanks to you, <CHARNAME>. Good to see you again. ~
 @1 = ~ And I am glad, high priest. Why did you send for me? ~
 @2 = ~ Sorry <CHARNAME> that I am bothering you again, but I have one assignment. And at that moment, you are the only one who can handle this. ~
-@3 = ~ I'm really sorry, but right now I don’t have time to work for you. Maybe later. ~
+@3 = ~ I'm really sorry, but right now I don't have time to work for you. Maybe later. ~
 @4 = ~ And what is it? ~
 @5 = ~ Far to the north of here in the frozen plains is an island, and under it should be a very valuable historical scroll. A hidden secret about the confrontation between Torm and Cyric. And yet - we learned that fans of Cyric sent their group of adventurers to find this scroll. ~
 @6 = ~ And what do we need to do for the high priest? ~
