--- PoFQuestPack\English\D\RANMMOD.tra	Mon Mar 16 19:53:46 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\RANMMOD.tra	Sun Mar 06 17:46:57 2022
@@ -11,9 +11,9 @@
 @11 = ~ Yes, there was a stranger here, he stopped only for one night, then he left. ~
 @12 = ~ And you do not know where he went? ~
 @13 = ~ I'm not interested in questions about every stranger if he leaves our village anyway. ~
-@14 = ~ This stranger we are talking about gathered an army of southerners, destroyed almost the entire cult of the Paladins of Faeríàn, killed many people, and even attacked the two cities of Faeríàn, Neverwinter and Targos. Our task is to find it and stop it, so if you know anything, tell us! ~
+@14 = ~ This stranger we are talking about gathered an army of southerners, destroyed almost the entire cult of the Paladins of Faerun, killed many people, and even attacked the two cities of Faerun, Neverwinter and Targos. Our task is to find it and stop it, so if you know anything, tell us! ~
 @15 = ~ My God, we treated him like a venerable paladin! Three days ago, I sent scouts north-east of our village. There are camps of orcs. There was never a big problem with them. And now, one of my scouts saw people among them, probably magicians. This is pretty unusual ... ~
-@16 = ~ Paladins of Faeríàn.
+@16 = ~ Paladins of Faerun.
 
 The Ranger Rolard saw Gradak, but he only stayed there one night, and then set off north of the village. There are camps of orcs. Scouts from the village saw that among them were magicians in red robes, whom we had already met once. Maybe he is collecting a ritual and wants to combine these 3 relics, we should get there as soon as possible. Rolard marked us this place on the map. ~
 @17 = ~ Hello to you from Gradak! ~
