--- PoFQuestPack\English\D\s#mage1.tra	Mon Mar 16 19:59:56 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\s#mage1.tra	Sun Mar 06 16:48:48 2022
@@ -2,4 +2,4 @@
 @1 = ~ Suppose this is so. Who are you? ~
 @2 = ~ I'm a mage from Neverwinter. Lord Nasher instructed me to track you down and deliver you to him as soon as possible. ~
 @3 = ~ Hey, wait, what does it mean to deliver?! ~
-@4 = ~ Don’t worry mi <LADYLORD>. Let's not waste time. Lord Nasher is waiting for you and will answer all your questions. ~
+@4 = ~ Don't worry mi <LADYLORD>. Let's not waste time. Lord Nasher is waiting for you and will answer all your questions. ~
