--- PoFQuestPack\English\D\SARMOD.tra	Mon Mar 16 19:59:56 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\SARMOD.tra	Sun Mar 06 16:48:48 2022
@@ -3,7 +3,7 @@
 @2 = ~ Killed, that's true. You showed me your power <CHARNAME>. ~
 @3 = ~ What do you want? ~
 @4 = ~ There is a very rare artifact in this city. The magic stone. My master ordered me to return to this world and get a stone for him. ~
-@5 = ~ Are you serving someone brother? I don’t understand. ~
+@5 = ~ Are you serving someone brother? I don't understand. ~
 @6 = ~ A lot has changed since you hit me. I have no choice. ~
 @7 = ~ So, you want to fight me again, brother? ~
 @8 = ~ What if I help you get rid of this host? We will fight side by side, like brothers?! ~
