--- PoFQuestPack\English\D\SCOUT1.tra	Mon Mar 16 19:59:56 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\SCOUT1.tra	Sun Mar 06 16:48:48 2022
@@ -1,4 +1,4 @@
 @1 = ~ How did it happen that you went around our traps and our patrols? This is not possible! ~
-@2 = ~ As you can see, it’s possible. Now tell me where can I find Gradak! ~
-@3 = ~ Gradak in the fortress, but you can’t get it ~
+@2 = ~ As you can see, it's possible. Now tell me where can I find Gradak! ~
+@3 = ~ Gradak in the fortress, but you can't get it ~
 @4 = ~ (At that moment you hear from the south how someone blows a horn.) ~
