--- PoFQuestPack\Russian\SETUP-PoFQuestPack.tra	Sun Mar 29 17:08:10 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\Russian\SETUP-PoFQuestPack.tra	Wed Jun 08 08:27:22 2022
@@ -2124,6 +2124,7 @@
 Требования:  Сила 18	
 Могут использовать:  -			
 ~
+@680 = ~~
 @681  = ~Привет!! Можете уделить мне минуточку?~
 @682  = ~Назначение~
 @683  = ~Назначение: Охрана при создании нового культа для защиты Фаэруна.~
