--- PoFQuestPack\English\D\SLAINE1.tra	Sun Mar 06 18:21:06 2022
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\SLAINE1.tra	Tue Mar 08 17:57:17 2022
@@ -1,4 +1,4 @@
-@1   = ~< CHARNAME > quickly here... They took him.~
+@1   = ~<CHARNAME> quickly here... They took him.~
 @2   = ~Wait, I don't know who took who?~
 @3   = ~Yarpen's warriors took my friend Solan...~
 @4   = ~Good.. Do you know where he is now?~
