--- PoFQuestPack\English\D\SRWZ1MOD.tra	Mon Mar 16 19:59:56 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\SRWZ1MOD.tra	Sun Mar 06 16:48:49 2022
@@ -1,4 +1,4 @@
-@0 = ~ Great job <CHARNAME>, thanks for the troll’s blood, but I will need one more thing. ~
+@0 = ~ Great job <CHARNAME>, thanks for the troll's blood, but I will need one more thing. ~
 @1 = ~ What exactly? ~
 @2 = ~ (The magician casts a spell and blood appears on his hands) ~
 @3 = ~ What is it? What did you do mage? ~
