--- PoFQuestPack\English\D\STRWZMOD.tra	Mon Mar 16 19:59:56 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\STRWZMOD.tra	Sun Mar 06 16:48:49 2022
@@ -10,7 +10,7 @@
 @9 = ~ Where can I find the prison they were taken to? ~
 @10 = ~ Even you <CHARNAME> cannot get there, you will have to kill a lot of innocent people, but this is not your style. ~
 @11 = ~ Show me on the map where this is? ~
-@12 = ~ I'm sorry, but I don’t know the exact location, you will have to look for help elsewhere. Now about why you came. ~
+@12 = ~ I'm sorry, but I don't know the exact location, you will have to look for help elsewhere. Now about why you came. ~
 @13 = ~ And what is it? ~
 @14 = ~ A group of vampires lives under our temple, so far we have lived in peace, but recently I had a magic wand stolen. ~
 @15 = ~ So we need to go down and bring this wand back? ~
@@ -24,7 +24,7 @@
 @21 = ~ Something I do not want, I do not want to risk my neck. Maybe later. ~
 @22 = ~ So you help us? ~
 @23 = ~ Yes, I'll see what can be done. ~
-@24 = ~ I don’t have time now. ~
+@24 = ~ I don't have time now. ~
 @25 = ~ Have you found these items? ~
 @26 = ~ Yes, there they are. ~
 @27 = ~ No, not yet. ~
