--- PoFQuestPack\English\D\TARO1MOD.tra	Mon Mar 16 20:06:24 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\TARO1MOD.tra	Sun Mar 06 16:48:49 2022
@@ -9,7 +9,7 @@
 
 My second task is to deal with the problems in the village north of the pass. ~
 @8 = ~ (false) I killed many creatures, but the Orc King Xangold escaped. ~
-@9 = ~ It’s a pity that Xangold escaped, but having lost a lot of troops, he may refuse to attack Targos. Here is your reward. I have a second task for you. ~
+@9 = ~ It's a pity that Xangold escaped, but having lost a lot of troops, he may refuse to attack Targos. Here is your reward. I have a second task for you. ~
 @10 = ~ Have you found the village and my friend? ~
 @11 = ~ I found them just in time, I saved all the rangers and your friend, he gave me this. ~
 @12 = ~ Yes, thanks, this is a certificate of goods in the village. You did a good job, here is your reward! ~
