--- PoFQuestPack\English\D\TAROMOD.tra	Mon Mar 16 20:06:24 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\TAROMOD.tra	Sun Mar 06 16:48:49 2022
@@ -7,7 +7,7 @@
 @6 = ~ As I said, we need a spy, you need to get as close to them as possible, find out what they are planning and prevent an attack on Targos. ~
 @7 = ~ I can take it. ~
 @8 = ~ I do not want to be your mercenary, and I hope you find someone else. ~
-@9 = ~ We don’t know exactly how many guards are around King Xangold, but he must be stopped. Penetrate into their camp and kill him. This will be your first task. I will lead you to the pass in the mountains, but then you will have to go alone. ~
+@9 = ~ We don't know exactly how many guards are around King Xangold, but he must be stopped. Penetrate into their camp and kill him. This will be your first task. I will lead you to the pass in the mountains, but then you will have to go alone. ~
 @10 = ~ Let's go. ~
 @11 = ~ War of the orcs.
 
