--- PoFQuestPack\English\D\THEGOCH.tra	Mon Mar 16 20:19:34 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\THEGOCH.tra	Sun Mar 06 17:46:57 2022
@@ -3,6 +3,6 @@
 @3 = ~ Yes, Gradak went this way. And he even mentioned you. You better get back to where you came from. You have no way to go through this passage! ~
 @4 = ~ I can't go back, I have to find Gradak. ~
 @5 = ~ In this case, you can just walk through the dungeon until you collect all 6 stones of the guards of the passage. In each part of the dungeon, one of them will be waiting for you. You must defeat them all or die. ~
-@6 = ~ Paladins of Faeríàn.
+@6 = ~ Paladins of Faerun.
 
 In the dungeon I met the guard of the passage. He said that Gradak did pass here, but in order to follow him, I will have to collect 6 stones of the guards of the passage. I will meet 6 guards in the dungeon and from each of them you need to get a protective stone. ~
