--- PoFQuestPack\English\D\VINTRELL.tra	Mon Mar 16 20:06:24 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\VINTRELL.tra	Sun Mar 06 16:48:49 2022
@@ -1,7 +1,7 @@
 @0 = ~ Wait a minute !! I ask you to help me! ~
 @1 = ~ Who are you and what do you want from me? ~
 @2 = ~ I am Wintrell, Torm's novice, I need help. ~
-@3 = ~ What’s the matter, why don’t you contact the mayor or local monastery for help? ~
+@3 = ~ What's the matter, why don't you contact the mayor or local monastery for help? ~
 @4 = ~ They have enough of their problems here, and I'm seriously injured, but I need to get to Torm's temple and report that something terrible happened! ~
 @5 = ~ What exactly? ~
 @6 = ~ Someone attacked a caravan heading from Torm Temple to Trademith, and all of ours were killed not far from here. Please find out who is responsible for this. ~
@@ -9,7 +9,7 @@
 @8 = ~ Unfortunately, I do not have time for this. Goodbye. ~
 @9 = ~ Order and Chaos - Additional POF storyline.
 
-I met the novice Torm Wintrell in Trademe, but I couldn’t help him. ~
+I met the novice Torm Wintrell in Trademe, but I couldn't help him. ~
 @10 = ~ Now, I marked this place on your map ~
 @11 = ~ I will tell you everything I can find out about it. ~
 @12 = ~ Order and Chaos - Additional POF storyline.
