--- PoFQuestPack\English\D\VIRA.tra	Mon Mar 16 20:06:24 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\VIRA.tra	Sun Mar 06 16:48:49 2022
@@ -2,6 +2,6 @@
 @2 = ~ I <CHARNAME>, you should have heard something about me ... ~
 @3 = ~ Yes, you are the legendary savior of Elminster from the valley of the icy wind! ~
 @4 = ~ Yes, and I'm here to kill you in the name of all the bad that you did for your Kuldahar house ... ~
-@5 = ~ I didn’t want to destroy the city, but there was no other way ... ~
+@5 = ~ I didn't want to destroy the city, but there was no other way ... ~
 @6 = ~ So you let them burn ... and kill all the people! You will regret the magician! ~
 @7 = ~ We will see! ~
