--- PoFQuestPack\English\D\WMLMOD.tra	Mon Mar 16 20:08:38 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\WMLMOD.tra	Sun Mar 06 16:48:49 2022
@@ -1,6 +1,6 @@
 @0 = ~ Who are you? ~
 @1 = ~ My name is <CHARNAME>, you sent for me. ~
-@2 = ~ Yeah, that’s you. I have a task for you. ~
+@2 = ~ Yeah, that's you. I have a task for you. ~
 @3 = ~ What is the task? ~
 @4 = ~ Find a strange magician for me, this is an unusual person, going around four cities again and again. He is a strange wizard, but he knows the secrets of a great treasure. He knows the exact location of the castle of the old king. According to legend, the castle was bewitched, because it revealed a dark evil. The castle is full of treasures. As a reward, you can leave everything found to yourself, with the exception of the equipment of the old king, bring it to me. ~
 @5 = ~ I'm sorry lord, but I don't have time to look for an old wizard. ~
