--- PoFQuestPack\English\D\WWCMOD.tra	Mon Mar 16 20:08:38 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\WWCMOD.tra	Sun Mar 06 16:48:49 2022
@@ -5,7 +5,7 @@
 @4 = ~ (smiles) I'm the captain of the guard, you are here because of Barbarosa. What do you know about him, what did you do there? ~
 @5 = ~ (false) I went there to have some fun, it's not forbidden. ~
 @6 = ~ The blacksmith Marian worked for Barbaros, I met him at Atkatla, he fled from here with a caravan. He asked me to deal with Barbarosa. ~
-@7 = ~ Don’t trick me buddy. I am your friend. Barbarossa is the enemy. So what did you do there? ~
+@7 = ~ Don't trick me buddy. I am your friend. Barbarossa is the enemy. So what did you do there? ~
 @8 = ~ (laughs) You just wanted to go to the city and deal with the most powerful black market boss in Deep Water? ~
 @9 = ~ Marian told me that you cannot help him, so he was looking for a mercenary or adventurer to help him. ~
 @10 = ~ Deep water obeys the secret lords, you may have already heard about this, some of them have secret informants in the trade guilds and the guild of thieves. ~
@@ -30,7 +30,7 @@
 We attacked Barbarossa, but he escaped before the battle began. ~
 @22 = ~ blacksmith Marian.
 
-I saved the blacksmith’s wife from the killers sent by Barbarosa, now I know all three passwords that are needed to enter the guild in the southeast of the city.
+I saved the blacksmith's wife from the killers sent by Barbarosa, now I know all three passwords that are needed to enter the guild in the southeast of the city.
 
 Password: slave, market, black. ~
 @23 = ~ Blacksmith Marian.
