--- PoFQuestPack\English\D\ZMOD.tra	Mon Mar 16 20:08:38 2020
+++ C:\BWP Patchstudio\patched files\PoFQuestPack\English\D\ZMOD.tra	Sun Mar 06 16:48:49 2022
@@ -5,9 +5,9 @@
 @4 = ~ Are you asking what I'm doing here? I heard that you have problems with the inhabitants of the local forest? ~
 @5 = ~ That I cannot allow you. To do this, you must obtain permission from the governor. ~
 @6 = ~ What about some gold coins for being able to talk to them? ~
-@7 = ~ It doesn’t concern you ... This problem will be solved soon! Anything else? ~
+@7 = ~ It doesn't concern you ... This problem will be solved soon! Anything else? ~
 @8 = ~ Hmmm. Okay, for 2,000 gold, I'll give you the camera keys. ~
-@9 = ~ I don’t have the right amount with me, I will come later. ~
+@9 = ~ I don't have the right amount with me, I will come later. ~
 @10 = ~ I just have the right amount, take it. ~
 @11 = ~ Here, keep the camera keys. ~
 @12 = ~ What, do you still want to pay for the conversation with the prisoners? ~
@@ -18,4 +18,4 @@
 @17 = ~ Tethyr.
 
 I received a letter from one of the prisoners, which must be delivered to the forest, but first I need to talk to the governor of the castle. ~
-@18 = ~ Don’t bother, go and talk to the governor, and I need to clean my weapons. ~
+@18 = ~ Don't bother, go and talk to the governor, and I need to clean my weapons. ~
