--- mih_eq\tob\amkethran\scripts\spawn_nature_elementals.baf	Thu Sep 30 21:32:46 2021
+++ C:\BWP Patchstudio\patched files\mih_eq\tob\amkethran\scripts\spawn_nature_elementals.baf	Sun Jan 09 16:01:25 2022
@@ -9,7 +9,7 @@
     SetGlobal("mh#SpawnNatureElementals", "AR5509", 1)
     ClearAllActions()
     StartCutSceneMode()
-    CreateCreature("cutspy", [710.750], S)
+    CreateCreature("cutspy", [710.750], 0)
     MoveViewPoint([737.568], 5)
     Wait(1)
     PlaySound("EFF_P92")
@@ -20,8 +20,8 @@
     Wait(1)
     CreateVisualEffect("spfleshs",[738.520])
     CreateVisualEffect("spfleshs",[724.672])
-    CreateCreature("mh#natel", [734.520], S)
-    CreateCreature("mh#natel", [724.672], S)
+    CreateCreature("mh#natel", [734.520], 0)
+    CreateCreature("mh#natel", [724.672], 0)
     ActionOverride("cutspy", DestroySelf())
     Wait(1)
     EndCutSceneMode()
