--- Setup-Oversight.tp2	Thu Jul 06 15:26:48 2017
+++ C:\BWP Patchstudio\patched files\Setup-Oversight.tp2	Sun Jun 09 18:29:14 2019
@@ -1,6 +1,6 @@
 BACKUP ~oversight/backup~
 AUTHOR "KishKelly@gmail.com"
-
+VERSION ~16 BWP Fix~
 README ~oversight/Oversight-ReadMe.docx~
 
 ALLOW_MISSING // ToB creatures
@@ -1443,15 +1443,15 @@
   REPLACE_TEXTUALLY ~.*\bTOTEMIC\b~                ~0x4010 TOTEMIC~
   REPLACE_TEXTUALLY ~.*\bSHAPESHIFTER\b~           ~0x4011 SHAPESHIFTER~
   REPLACE_TEXTUALLY ~.*\bBEASTFRIEND\b~            ~0x4012 BEASTFRIEND~
-  REPLACE_TEXTUALLY ~.*\bBARBARIAN\b~              ~0x0000 BARBARIAN~
-  REPLACE_TEXTUALLY ~.*\bWILDMAGE\b~               ~0x0000 WILDMAGE~
+  REPLACE_TEXTUALLY ~.*\bBARBARIAN\b~              ~0x40000000 BARBARIAN~
+  REPLACE_TEXTUALLY ~.*\bWILDMAGE\b~               ~0x80000000 WILDMAGE~
   BUT_ONLY_IF_IT_CHANGES
 
 APPEND ~kit.ids~ ~0x4000 TRUECLASS~
   UNLESS ~\bTRUECLASS\b~
-APPEND ~kit.ids~ ~0x0000 BARBARIAN~
+APPEND ~kit.ids~ ~0x40000000 BARBARIAN~
   UNLESS ~\bBARBARIAN\b~
-APPEND ~kit.ids~ ~0x0000 WILDMAGE~
+APPEND ~kit.ids~ ~0x80000000 WILDMAGE~
   UNLESS ~\bWILDMAGE\b~
 
 // force WeiDU to reload ids files
