--- FotD\BAF_NEW\CUCHOL.baf	Mon Nov 28 08:58:54 2016
+++ C:\BWP Patchstudio\patched files\FotD\BAF_NEW\CUCHOL.baf	Tue Nov 29 08:09:02 2016
@@ -184,7 +184,7 @@
 END
 
 IF
-	HaveSpell(WIZARD_FEAR)
+	HaveSpell(WIZARD_HORROR)
 	!General(TenthNearestEnemyOf(Myself),UNDEAD)
 	!StateCheck(TenthNearestEnemyOf(Myself),STATE_CONFUSED)
 	!StateCheck(TenthNearestEnemyOf(Myself),STATE_FEEBLEMINDED)
@@ -199,7 +199,7 @@
 END
 
 IF
-	HaveSpell(WIZARD_FEAR)
+	HaveSpell(WIZARD_HORROR)
 	!General(NinthNearestEnemyOf(Myself),UNDEAD)
 	!StateCheck(NinthNearestEnemyOf(Myself),STATE_CONFUSED)
 	!StateCheck(NinthNearestEnemyOf(Myself),STATE_FEEBLEMINDED)
@@ -214,7 +214,7 @@
 END
 
 IF
-	HaveSpell(WIZARD_FEAR)
+	HaveSpell(WIZARD_HORROR)
 	!General(EighthNearestEnemyOf(Myself),UNDEAD)
 	!StateCheck(EighthNearestEnemyOf(Myself),STATE_CONFUSED)
 	!StateCheck(EighthNearestEnemyOf(Myself),STATE_FEEBLEMINDED)
@@ -229,7 +229,7 @@
 END
 
 IF
-	HaveSpell(WIZARD_FEAR)
+	HaveSpell(WIZARD_HORROR)
 	!General(SeventhNearestEnemyOf(Myself),UNDEAD)
 	!StateCheck(SeventhNearestEnemyOf(Myself),STATE_CONFUSED)
 	!StateCheck(SeventhNearestEnemyOf(Myself),STATE_FEEBLEMINDED)
@@ -244,7 +244,7 @@
 END
 
 IF
-	HaveSpell(WIZARD_FEAR)
+	HaveSpell(WIZARD_HORROR)
 	!General(SixthNearestEnemyOf(Myself),UNDEAD)
 	!StateCheck(SixthNearestEnemyOf(Myself),STATE_CONFUSED)
 	!StateCheck(SixthNearestEnemyOf(Myself),STATE_FEEBLEMINDED)
@@ -259,7 +259,7 @@
 END
 
 IF
-	HaveSpell(WIZARD_FEAR)
+	HaveSpell(WIZARD_HORROR)
 	!General(FifthNearestEnemyOf(Myself),UNDEAD)
 	!StateCheck(FifthNearestEnemyOf(Myself),STATE_CONFUSED)
 	!StateCheck(FifthNearestEnemyOf(Myself),STATE_FEEBLEMINDED)
@@ -274,7 +274,7 @@
 END
 
 IF
-	HaveSpell(WIZARD_FEAR)
+	HaveSpell(WIZARD_HORROR)
 	!General(FourthNearestEnemyOf(Myself),UNDEAD)
 	!StateCheck(FourthNearestEnemyOf(Myself),STATE_CONFUSED)
 	!StateCheck(FourthNearestEnemyOf(Myself),STATE_FEEBLEMINDED)
@@ -289,7 +289,7 @@
 END
 
 IF
-	HaveSpell(WIZARD_FEAR)
+	HaveSpell(WIZARD_HORROR)
 	!General(ThirdNearestEnemyOf(Myself),UNDEAD)
 	!StateCheck(ThirdNearestEnemyOf(Myself),STATE_CONFUSED)
 	!StateCheck(ThirdNearestEnemyOf(Myself),STATE_FEEBLEMINDED)
@@ -304,7 +304,7 @@
 END
 
 IF
-	HaveSpell(WIZARD_FEAR)
+	HaveSpell(WIZARD_HORROR)
 	!General(SecondNearestEnemyOf(Myself),UNDEAD)
 	!StateCheck(SecondNearestEnemyOf(Myself),STATE_CONFUSED)
 	!StateCheck(SecondNearestEnemyOf(Myself),STATE_FEEBLEMINDED)
@@ -319,7 +319,7 @@
 END
 
 IF
-	HaveSpell(WIZARD_FEAR)
+	HaveSpell(WIZARD_HORROR)
 	!General(NearestEnemyOf(Myself),UNDEAD)
 	!StateCheck(NearestEnemyOf(Myself),STATE_CONFUSED)
 	!StateCheck(NearestEnemyOf(Myself),STATE_FEEBLEMINDED)
@@ -334,7 +334,7 @@
 END
 
 IF
-	HaveSpell(WIZARD_FEAR)
+	HaveSpell(WIZARD_HORROR)
 	!Allegiance(LastSeenBy(Myself),ENEMY)
 	!General(LastSeenBy(Myself),UNDEAD)
 	!StateCheck(LastSeenBy(Myself),STATE_CONFUSED)
@@ -348,7 +348,7 @@
 THEN
 	RESPONSE #100
 		SetGlobalTimer("CastAndAttack","LOCALS",6)
-		Spell(LastSeenBy(Myself),WIZARD_FEAR)
+		Spell(LastSeenBy(Myself),WIZARD_HORROR)
 END
 
 IF
