--- TDD\Language\French\POCRE001.tra	Mon Nov 09 15:08:08 2009
+++ C:\BWP Patchstudio\patched files\TDD\Language\French\POCRE001.tra	Mon Dec 25 10:11:21 2017
@@ -16,3 +16,5 @@
 @15    = ~Etes-vous prêt à quitter le lac ?~ ~Etes-vous prête à quitter le lac ?~
 @16    = ~Oui, faites-moi partir d'ici.~
 @17    = ~Non, pas encore.~
+@18   = ~Le lac~
+@19   = ~L'Auberge de Sunlake~
\ No newline at end of file
