--- setup-TenyaThermidor.tp2	Sun Feb 08 19:50:19 2015
+++ C:\BWP Patchstudio\patched files\setup-TenyaThermidor.tp2	Tue Nov 29 09:26:52 2016
@@ -2,7 +2,7 @@
 // Basics
 BACKUP ~TenyaThermidor/backup~
 AUTHOR ~Coutelier (mail@coutelier.org.uk)~
-
+VERSION ~v1.5 BWP conversion~
 // --------------------------------------------
 // BG Tenya Thermidor v1.5
 // --------------------------------------------
@@ -16,7 +16,7 @@
          ~TenyaThermidor/tra/english/TTsetup.tra~
 
 // Component setup
-BEGIN ~Tenya Thermidor for BG:EE~
+BEGIN ~Tenya Thermidor for BWP~
 
 //Spells
 ADD_SPELL "TenyaThermidor/Spells/B2#TNSTM.spl" 4 0 B2#TNSTM
